test/test_quantized_models.py

```diff
@@ -4,26 +4,47 @@
 from common_quantization import QuantizationTestCase, ModelMultipleOps
```

Comment:
why is this not in test_quantization.py?

Reply:
this looks like an integration test

Reply:
Let's chat. test_quantized_models covers tests at a higher level: more complex models and comparison of float with quantized numerics at the model level. I think its better to have it as a separate file as test_quantization is really large already